omero/users/clients-overview.txt

```diff
@@ -16,10 +16,10 @@ installations worldwide, OMERO has been shown to be relatively easy to install
 and get running.
 
 OMERO.insight and OMERO.importer are desktop applications written in Java
-and require Java 1.6 (or higher) to be installed on the user's computer
+and require Java 1.7 (or higher) to be installed on the user's computer
 (automatically installed on most up-to-date OS X and Windows XP systems).
 
-Our **NEW** user assistance :help:`help website<>` provides a series of 
+Our user assistance :help:`help website<>` provides a series of 
```

Comment:
I looked through 5.1 refs yesterday and didn't see anything that needed removing, although I will double-check with that command and maybe rephrase to 'introduced in 5.1' or similar.